docs/getting_started/get_started.md

```diff
@@ -141,4 +141,8 @@ Then, click **Change password**.
 Fill in all the required fields and click **Save and close** to save changes.
 Click **Discard** to reject your changes and return to the previous screen.
 
-![Editing password](img/editing_password.png "Editing password")
\ No newline at end of file
+![Editing password](img/editing_password.png "Editing password")
+
+**Check out the interactive demo below:**
```

Comment:
Again - you've used this phrase in one other place, and not in the others - why not skip it here, too?